### PR TITLE
SONARIAC-811 Rule S6364: ARM Defining a short backup retention duration is security-sensitive - WebSites

### PR DESCRIPTION
**Pull request by rudy-regazzoni-sonarsource**

### CODE REVIEW

File: iac-extensions/arm/src/test/resources/checks/ShortBackupRetentionCheck/Microsoft.Web_sites_custom.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      "type": "Microsoft.Web/sites/config",
+      "apiVersion": "2022-03-01",
+      "properties": {
+        "backupSchedule": {
+          "retentionPeriodInDays": 20
+        }
+      }
+    }
+  ]
+}
```
Comment (mstachniuk) [inline iac-extensions/arm/src/test/resources/checks/ShortBackupRetentionCheck/Microsoft.Web_sites_custom.json:27]: New lines in JSON files at the end

**Review by mstachniuk — COMMENTED** [review]
Please see my question/concern

File: iac-extensions/arm/src/test/resources/checks/ShortBackupRetentionCheck/Microsoft.Web_sites.json [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        }
+      }
+    },
+    {
+      "name": "Sensitive: sensitive case in nested resource",
+      "type": "Microsoft.Web/sites",
+      "apiVersion": "2022-03-01",
+      "resources": [
+        {
+          "name": "Nested child",
```
[the remark this answers is not in the document]
Comment (rudy-regazzoni-sonarsource) [reply iac-extensions/arm/src/test/resources/checks/ShortBackupRetentionCheck/Microsoft.Web_sites.json:41]: Damn I checked, apparently it's a specific case where the name does care and changes the property/definition of the resource.
That's very odd and ugly IMO, but definitely it means we should check this.
Good catch!

**Review by mstachniuk — APPROVED** [review]
LGTM

**Pull request merged**